pkg/controller/container-runtime-config/container_runtime_config_controller.go

```diff
@@ -618,6 +626,15 @@ func (ctrl *Controller) syncContainerRuntimeConfig(key string) error {
 		return ctrl.syncStatusOnly(cfg, err)
 	}
 
+	// create the MC for the drop in default-container-runtime crio.conf file
+	if err := ctrl.createDefaultContainerRuntimeMC(cfg); err != nil {
```

Comment:
we would not get a cfg object if the `key` passed to `syncContainerRuntimeConfig` is `forceSyncOnUpgrade`. `syncContainerRuntimeConfig` would return before this line because the ContainerRuntimeConfig does not exist.

Reply:
Oh, so the best way here is to query an API and get the cfg?

Reply:
I just think if we have to fetch the machineconfigs to check if the existing runtime configuration has been set on the cluster we don't neet the cfg argument for `createDefaultContainerRuntimeMC`